gen/calc.py

```diff
@@ -420,6 +420,64 @@ def calculate_cluster_docker_registry_enabled(cluster_docker_registry_url):
     return 'false' if cluster_docker_registry_url == '' else 'true'
 
 
+def validate_cosmos_config(cosmos_config):
+    """The schema for this configuration is.
+    {
+      "schema": "http://json-schema.org/draft-04/schema#",
+      "type": "object",
+      "properties": {
+        "staged_package_storage_uri": {
+          "type": "string"
+        },
+        "package_storage_uri": {
+          "type": "string"
+        }
+      }
+    }
+    """
+
+    config = json.loads(cosmos_config)
```

Comment:
Please use https://github.com/dcos/dcos/blob/master/gen/calc.py#L187 to load out this dictionary, makes nice errors if the user provides something not json